gcp/modules/locust/variables.tf

```diff
@@ -1,13 +1,13 @@
 variable "locust_script" {
-  default = "preferences.py"
+  default = ""
 }
 
 variable "locust_target_host" {
   default = ""
 }
 
 variable "locust_swarm_duration" {
-  default = 60
+  default = 50
```

Comment:
Right, 50 means 60 because we add 10 to it later.

It's fine the way you've implemented it but you don't think this is weird? Do you think it warrants a comment?

Reply:
Ok, let's just exit if `locust_swarm_duration` is unset.